src/services/sqs.ts

```diff
@@ -14,7 +16,8 @@ export const addToSqs = async (messageBody: SQSRequestBody, queueUrl: string) =>
   };
 
   try {
-    await sqsClient.send(new SendMessageCommand(params));
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-argument
+    await sqsClient.send(new SendMessageCommand(params) as any);
```

Comment:
why do you need to cast it as any?

Reply:
@kcarr-bjss why is this changed like this? I didn't do this bit so

Reply:
donezo